File: code-coaches/easy/number-of-ones/NumberOfOnes.cs
using System;

public class NumberOfOnes
{
    static void Main(string[] args)
    {
        int num = int.Parse(Console.ReadLine());
        string numBin = Convert.ToString(num, 2);
        int numOfOnesInBin = numBin.Split('1').Length - 1;

        Console.WriteLine(numOfOnesInBin);
    }
}

File: code-coaches/easy/number-of-ones/NumberOfOnes.swift
func main() {
 	let intNum = Int(readLine()!)!
	let numBin = String(intNum, radix: 2)
	let numOfOnesInBin = numBin.filter { $0 == "1" }.count
	
	print(numOfOnesInBin)
}

main()